ESPixelStick/src/network/WiFiDriver.cpp

```diff
@@ -31,13 +31,15 @@
 #include "../FileMgr.hpp"
 
 //-----------------------------------------------------------------------------
-// Create secrets.h with a #define for SECRETS_SSID and SECRETS_PASS
+// Use platformio_user.ini to define these values.
+// Platformio_user.ini defines these in the [env] section.
 // or delete the #include and enter the strings directly below.
-#include "secrets.h"
-#ifndef SECRETS_SSID
+#if !defined(SECRETS_SSID)
 #   define SECRETS_SSID "DEFAULT_SSID_NOT_SET"
+#endif // SECRETS_SSID
+#if !defined(SECRETS_PASS)
 #   define SECRETS_PASS "DEFAULT_PASSPHRASE_NOT_SET"
-#endif // ndef SECRETS_SSID
```

Comment:
Please do NOT change the endif strings. They tails are there on purpose.

Reply:
fixed typo